doc/ansible/information-flow-ansible-operator.md

```diff
@@ -24,8 +24,9 @@
 
 ### The Proxy
  * Every request to the k8s api goes through the proxy.
- * The owner reference is injected into the object that is being created.
- * The proxy then adds dependent watches for the correct controller if we have not started watching the type already
+ * The owner reference is injected into the object that is being created in the same namespace as the CR.
+ * The operator-sdk annotations are injected into the object that is being created outside of namepsace of the CR.
```

Comment:
I am not sure if it makes sense for me 
What `operator-sdk annotations` are you speaking about here?

Reply:
The operator sdk annotations are: `operator-sdk/primary-resource` and `operator-sdk/primary-resource-type`

These are applied to k8s resources that are created in a different namespace to the namespace that the Custom Resource is created in, becuase the owner references for garbage collection wouldn't work between namespaces.